notebooks/lst_sensitivity_pyirf.ipynb

```diff
@@ -0,0 +1,1726 @@
+{
```

Comment:
<p>Add brief explanation for the 0.002 and the dead time per event?</p><p>I think the latter is (in principle at least) 7 microsecs, but please check it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Seems that I didn't pushed the final version of this funcion, sorry. It is corrected now. The dead time per event is not read from deltat (and effectively is ~7 microseconds). Now I changed the deltat limit to be &lt; 0.1 seconds, which is the value suggested in the analysis call., to exclude big jumps between runs. </p>

Reply:
<p>Isn't this function also in lstchain now and could just be imported?</p>

Reply:
<p>Yes! not merged yet though. Once it is merged, I will update the notebook. </p>